tests/utils.go

```diff
@@ -128,11 +128,13 @@ const MigrationWaitTime = 240
 const ContainerCompletionWaitTime = 60
 
 func TestCleanup() {
+	GinkgoWriter.Println("Global test cleanup started.")
```

Comment:
Usually we use `By` for this purpose. What is the advantage of `GinkgoWriter.Println` over `By`?

Reply:
The documentation states that `By` purpose is to split a test into blocks.
I thought it is less of a fit in this case. My interpretation of tge docs is that it fits in the tests (It).